helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterStatusMonitor.java

```diff
@@ -1121,4 +1120,13 @@ public long getPendingStateTransitionGuage() {
     }
     return total;
   }
+
+  @Override
+  public long getRebalanceThrottledByErrorPartition() {
+    long total = 0;
+    for (Map.Entry<String, ResourceMonitor> entry : _resourceMonitorMap.entrySet()) {
```

Comment:
i am not great with Java8 lambda, but can this be written using that??  Again, I am not expert.

Reply:
IMHO, lamda is slower. Probably we want better performance in functions being called rapidly. (1/min)

Reply:
Performance wise, Lambda/stream is not necessarily slower than for-loop, we also have parallel stream. Besides, benchmarking and then optimizing is the usual way for performance optimization.

I don't have strong opinion on either approach, but if you go with forloop, you can simply use `Map.values()` if you don't care about keys.

Reply:
TFTR. I slightly would prefer keep it homogeneous to other functions and the I could run a small profiler later. Could do optimization for all these metrics aggregation functions later in a separate PR.

Reply:
This is a simple addition and no exceptions are thrown so lamda will be a good option. But also agree that we can keep the style consistent throughout the codebase and do a global replacement later once we know more about the performance impact.